change_note_duration.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import fileinput
+import itertools
+import os
+import sys
+
+parser = argparse.ArgumentParser(description='Double or halve the note times in a file. '
+                                             'This does NOT change the time signature.',
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('file', help="Path to file where note values will be changed")
+parser.add_argument('-start_delimiter',
+                    default='Music',
+                    help='A string to search for in the file to indicate where note values will begin.')
+parser.add_argument('-end_delimiter',
+                    default='Verses',
+                    help='A string to search for in the file to indicate where note values end.')
+parser.add_argument('--double', action='store_true', help="Double note duration instead of halving it.")
+
+args = parser.parse_args()
+
+
+class NoteValueError(Exception):
+    pass
+
+
+def get_music_and_lyrics_line_numbers(file_name, start_delimiter, end_delimiter):
+    music_line = None
+    lyrics_line = None
+    with open(file_name, 'r') as file:
+        for (i, line) in enumerate(file):
+            if not music_line and start_delimiter in line:
+                music_line = i
+            if not lyrics_line and end_delimiter in line:
+                lyrics_line = i
+    return (music_line, lyrics_line,)
+
+
+def halve_note_duration(char):
+    try:
+       return str(int(char)*2)
+    except ValueError:
+        return char
+
+def double_note_duration(char):
```

Comment:
...same here.

Oh also: the `try`s will always succeed because you're only calling these functions on the groups where `str.isdigit` is true.